docs/tutorials/blogs.md

```diff
@@ -0,0 +1,1087 @@
+
+        The tags plugin in the Insider Edition is an incredibly powerful tool
+        and we can only scratch the surface of what is possible with it. If you
+        want to explore more after you have worked for this part of the tutorial,
+        have a look at the [tags plugin reference].
+
+[tags plugin reference]: ../plugins/tags.md
+
+## 10. Pagination
+
+Once your blog starts growing, you may not want to pay attention to the number
+of posts displayed per page. By default, the plugin displays up to 10 posts on
+the archive and
+
+
+
+
+## 11. Social Media
+
+The blog plugin works well with the social plugin, which creates social cards
+for all pages, including blog posts. Social cards are images that other systems
+such as social media can display as a preview for content linked to.
+It is easy to get started with the social plugin, true to the motto of
+Material with MkDocs: "batteries included."
+
+!!! example "Add social cards"
+
+    To add social cards to your blog you need to install some dependencies.
+    These differ depending on what operating system you use.
+
+    TODO: paste in installation instructions here - use snippets since this is
+    a lot of duplicated material?
+
+    Once these prerequisites are fulfilled, you can simply add the social plugin
+    to your list of plugins:
+
+    ```yaml hl_lines="10"
+        plugins:
+            - search
+            - blog:
+                authors_profiles: true
+                categories_allowed:
+                    - Holidays
+                    - News
+            - tags:
+                tags_file: blog/tags.md
+            - social
+    ```
+
+    Now, then you run `mkdocs build` and look at the `site` directory, you will
+    see that it contains subfolders under `assets/images/social` that reflect
+    the structure of your Markdown files. Each page has a corresponding PNG file
+    that contains the social card image. Of interest here are the social cards
+    for the blog posts, so have a look at them.
+
+    TODO: posting them to social media - demo with Mastodon?
+
+Now, the social plugin provides a number of default layouts and configuration
+options for changing aspects such as colors, images, fonts, logos, the title,
+even the description.
+
+
+!!! example "Changing background color"
+
+    For example, to set the background color to an attention-grabbing hot pink,
+    you might add:
+
+    ```yaml hl_lines="4 5"
+    plugins:
+    ...
+    - social:
+        cards_layout_options:
+            background_color: "#ff1493"
+    ```
+
+!!! tip "Override card layout in page header <!-- md:sponsors -->"
+
+    With the Insider Edition, you can customize the card layout for each
+    page by adding settings to the page header. So, you want only certain
+    cards to be in hot pink:
+
+    ```yaml
+    ---
+    social:
+      cards_layout_options:
+        background_color: "#ff1493"
+    ---
+    ```
+
+### Custom cards <!-- md:sponsors -->
```

Comment:
Custom social cards should be a tutorial of its own. Please remove this section.